lib/improver/tests/ensemble_calibration/ensemble_calibration/helper_functions.py

```diff
@@ -148,12 +149,15 @@ def set_up_probability_above_threshold_spot_temperature_cube():
             data, "air_temperature", "degreesC"))
 
 
-def set_up_cube(data, standard_name=None, units=None, long_name=None,
-                realizations=np.array([0, 1, 2]), timesteps=1,
-                y_dimension_length=3, x_dimension_length=3):
+def set_up_cube(data, name, units, realizations=np.array([0, 1, 2],
+                dtype=np.float32), timesteps=1, y_dimension_length=3,
+                x_dimension_length=3):
```

Comment:
What is wrong with the realization points being integers here?

Reply:
Done.